### ISSUE TITLE
Website is down

### ISSUE THREAD

**Issue by bendem** [shieldproject/shield#756]
I just came across a recommendation for Shield, but the [website](https://www.shieldproject.io/) seems down. Is there any documentation as to what this project does and how?

**Comment by norman-abramovitz** [comment]
As we work through the issues with the company that owns the dns entry, your best bet is to read the github Readmes or on slack.

**Comment by joestump** [comment]
@norman-abramovitz is this project still maintained?

**Comment by norman-abramovitz** [comment]
Yes, the code is still maintained and improved and used..  Give me a couple days, since I  just came home from some major thoracic surgery.   My head is not clear enough to remember where the forked was done with the location and where to find the updated artifacts.

**Comment by joestump** [comment]
@norman-abramovitz oh man, sorry to hear! Wishing you a speedy recovery ❤ If I can be helpful, let me know.

**Comment by krutten** [comment]
We moved to https://shieldproject.cloud

I will go through the repos and try and find any references not updated and update them.  We have some update in our sprint backlog for version bumps and updates, but no ETA at this moment.

**Comment by norman-abramovitz** [comment]
Try https://shieldproject.cloud/  It is not perfect, but a start.

**Comment by norman-abramovitz** [comment]
@joestump See previous message.   back to not laughing, it hurts too much! 😄

**Comment by wayneeseguin** [comment]
Closing this as it appears to be resolved.
